modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad/src/main/java/com/liferay/dynamic/data/mapping/uad/exporter/DDMFormInstanceUADExporter.java

```diff
@@ -14,13 +14,53 @@
 
 package com.liferay.dynamic.data.mapping.uad.exporter;
 
+import com.liferay.dynamic.data.mapping.service.DDMFormInstanceRecordLocalService;
+import com.liferay.portal.kernel.dao.orm.ActionableDynamicQuery;
+import com.liferay.portal.kernel.dao.orm.DynamicQuery;
+import com.liferay.portal.kernel.dao.orm.ProjectionFactoryUtil;
+import com.liferay.portal.kernel.dao.orm.Property;
+import com.liferay.portal.kernel.dao.orm.PropertyFactoryUtil;
+import com.liferay.portal.kernel.dao.orm.RestrictionsFactoryUtil;
 import com.liferay.user.associated.data.exporter.UADExporter;
 
 import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
 
 /**
  * @author Brian Wing Shun Chan
  */
 @Component(immediate = true, service = UADExporter.class)
 public class DDMFormInstanceUADExporter extends BaseDDMFormInstanceUADExporter {
+
+	@Override
+	protected ActionableDynamicQuery getActionableDynamicQuery(long userId) {
+		ActionableDynamicQuery actionableDynamicQuery =
+			doGetActionableDynamicQuery();
+
+		actionableDynamicQuery.setAddCriteriaMethod(
+			dynamicQuery -> {
+				DynamicQuery formInstanceIdDynamicQuery =
+					_ddmFormInstanceRecordLocalService.dynamicQuery();
```

Comment:
@marcosapmf, should we use `DDMFormInstanceRecordLocalService` or `DDMFormInstanceLocalService` here? Just let me know your answer and I do this change locally.

Thanks.